picky/src/x509/pkcs7/mod.rs

```diff
@@ -3,12 +3,14 @@ use crate::x509::certificate::{Cert, CertError};
 use crate::x509::utils::{from_der, from_pem, from_pem_str, to_der, to_pem};
 use crate::AlgorithmIdentifier;
 use picky_asn1_der::Asn1DerError;
-use picky_asn1_x509::content_info::EncapsulatedContentInfo;
 use picky_asn1_x509::pkcs7::Pkcs7Certificate;
 use picky_asn1_x509::signed_data::CertificateChoices;
-use picky_asn1_x509::signer_info::SignerInfo;
 use thiserror::Error;
 
+pub use picky_asn1_x509::algorithm_identifier;
+pub use picky_asn1_x509::content_info::EncapsulatedContentInfo;
+pub use picky_asn1_x509::signer_info::SignerInfo;
+
```

Comment:
issue: Let’s not change the public API of the `picky` crate in this PR. You can add a dependency on `picky-asn1-x509` in the `picky-ffi` crate instead.

Reply:
updated